Update dg setcrs (#294)
AY2324S2-CS2103T-F13-1/tp: docs/DeveloperGuide.md

| {{ threeStars }} | TA {{ whitespace }} | name/rename the CS course that I am tutoring this semester                  | keep track of the module I am teaching         |

> | {{ threeStars }} | TA                  | add a student to the CS course that I am tutoring that semester to my class | keep track of him or her                       |

Reviewer: The wording can be improved:
```suggestion
| {{ threeStars }} | TA                  | add a student to my class that I am tutoring this semester | keep track of him or her                       |
```

| {{ threeStars }} | TA                  | view all students from my class                                             | view details about all of them                 |

> | {{ threeStars }} | TA                  | mark attendance for a student in my class for a particular week             | keep track of who's present                    |
> | {{ threeStars }} | TA                  | unmark attendance for a student in my class for a particular week           | keep track of who is absent                    |

Reviewer: Perhaps we can be more consistent with our phrasing:
```suggestion
| {{ threeStars }} | TA                  | mark attendance for a student in my class for a particular week             | keep track of who is present                    |
| {{ threeStars }} | TA                  | unmark attendance for a student in my class for a particular week           | keep track of who is absent                    |
```

| {{ threeStars }} | TA                  | delete a student                                                            | remove a student if he or she leaves the class |
| {{ threeStars }} | TA                  | know all the commands of TAPro via the help window                          | use it effectively                             |
| {{ threeStars }} | TA                  | see all students in the contact book                                        | have an overview of all students               |
| {{ threeStars }} | TA                  | edit a student's details                                                    | have the latest data                           |
| {{ threeStars }} | TA                  | find a student by name                                                      | get a student's data easily                    |

> | {{ threeStars }} | TA                  | delete all students from the contact book                                   | reset my student's data quickly                |

Reviewer: We should be more specific with this user story.
```suggestion
| {{ threeStars }} | TA                  | delete all students from a previous semester from the contact book                                   | clear my contacts quickly at the start of a semester                |
```

| {{ twoStars }}   | TA                  | retrieve command history                                                    | avoid retyping a command                       |

> | {{ twoStars }}   | TA                  | use TAB to autocomplete my input                                            | to save time                                   |

Reviewer: We should not be too specific in user stories about how a feature is implemented
```suggestion
| {{ twoStars }}   | TA                  | to autocomplete my input                                            | to save time                                   |
```
